### ISSUE TITLE
Using junit-vintage-engine (5.6.0)

### ISSUE THREAD

**Issue by khmarbaise** [sormuras/junit-platform-maven-plugin#36]
I'm trying to use the vintage-engine in a project in combination with JUnit JUpiter Tests etc. and getting the following:

```
[INFO] --- junit-platform-maven-plugin:1.0.0-M5:launch (injected-launch) @ junit-jupiter ---
[INFO] Launching JUnit Platform 1.6.0...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.261 s
[INFO] Finished at: 2020-02-26T17:26:01+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal de.sormuras.junit:junit-platform-maven-plugin:1.0.0-M5:launch (injected-launch) on project junit-jupiter: Execution failed!: java.lang.NoClassDefFoundError: org/junit/platform/engine/TestEngine: org.junit.platform.engine.TestEngine -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

Might this be related to the upgrade issue?
And yes I have added the vintage as dependency:
```xml
    <dependency>
      <groupId>org.junit.vintage</groupId>
      <artifactId>junit-vintage-engine</artifactId>
      <scope>test</scope>
    </dependency>
```

**Comment by sormuras** [comment]
Strange. Using Jupiter-only worked before?

**Comment by sormuras** [comment]
Should be fixed with `junit-platform-maven-plugin:1.0.0` -- can please retest?
